### ISSUE TITLE
when is bootstrap 4 going to relase

### ISSUE THREAD

**Issue by sailajasmart** [twbs/bootstrap#22726]
Before opening an issue:

- [Search for duplicate or closed issues](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [Validate](https://validator.w3.org/nu/) and [lint](https://github.com/twbs/bootlint#in-the-browser) any HTML to avoid common problems
- Prepare a [reduced test case](https://css-tricks.com/reduced-test-cases/) for any bugs
- Read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/master/CONTRIBUTING.md)

When asking general "how to" questions:

- Please do not open an issue here
- Instead, ask for help on [StackOverflow, IRC, or Slack](https://github.com/twbs/bootstrap/blob/master/README.md#community)

When reporting a bug, include:

- Operating system and version (Windows, Mac OS X, Android, iOS, Win10 Mobile)
- Browser and version (Chrome, Firefox, Safari, IE, MS Edge, Opera 15+, Android Browser)
- Reduced test cases and potential fixes using [JS Bin](https://jsbin.com)

When suggesting a feature, include:

- As much detail as possible for what we should add and why it's important to Bootstrap
- Relevant links to prior art, screenshots, or live demos whenever possible

**Comment by Johann-S** [comment]
When everything here : https://github.com/twbs/bootstrap/issues?q=is%3Aopen+is%3Aissue+milestone%3Av4.0.0-beta will be closed/resolved

**Comment by Retroity** [comment]
Bootstrap 4 is currently at Alpha 6 and is currently being worked on. It's not even at Beta 1 yet. You can see what's being done to move towards Beta 1 here: https://github.com/twbs/bootstrap/projects/3

As you can see, there's still a lot to be done. According to the v4 Alpha page, Bootstrap v4 is a major rewrite of nearly the entire project. (from here: https://v4-alpha.getbootstrap.com/migration/)

Basically, when everything that needs to be done is completed.

**Comment by rafalp** [comment]
Also look at mdo's response to this very question from yesterday:

https://news.ycombinator.com/item?id=14446066
